reana-ui/src/components/Error.js

```diff
@@ -0,0 +1,35 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import React from "react";
+import { Container, Image } from "semantic-ui-react";
+import PropTypes from "prop-types";
+
+import LogoImg from "../images/logo-reana.svg";
+
+import styles from "./Error.module.scss";
+
+export default function Error({ message }) {
+  return (
+    <Container textAlign="center" className={styles.container}>
+      <Image centered spaced src={LogoImg} size="small" />
+      <h1>Error</h1>
```

Comment:
Everything works nicely, I have just two small comments which might not be relevant, I leave it for your interpretation and @tiborsimko's:
![image](https://user-images.githubusercontent.com/9035606/82799036-ae41ca00-9e79-11ea-83d3-674eaeb1b917.png)
_Screenshot to illustrate_
- The word error is very big compared to REANA logo (an impression which might be caused by the second point)
- Logo and "Error" (negative) word are very near to each other

Reply:
This was intended to be a first minimal version of error pages, there's a lot of room for improvement. I just wanted to avoid returning a plain text "Forbidden".

Reply:
Yeah, it's not really an "internal error" kind of situation, more an "information passing" kind of situation, which the  current mock-up may not evoke very well graphically. Two suggestions: instead of "Error" you could say "Access denied"; the informative message could say something more verbose like "Sorry, the user access token aaaa-bbbb-1111-2222 was revoked on YYYY-MM-DD hh:mm:ss and cannot be used anymore."  Otherwise fully agree, we can improve UI later for all the various error/warning/info kind of situations we might have.

Reply:
I've changed the title and made it smaller for now.

![image](https://user-images.githubusercontent.com/8863238/82814262-1b178d00-9e97-11ea-9490-8df12bb0589e.png)

I'll create a ticket to further improve error pages. We might get more inspiration when we have more cases.